drivers/npmx/npmx_shell.c

```diff
@@ -1649,6 +1658,330 @@ static int cmd_ldsw_ldo_voltage_get(const struct shell *shell, size_t argc, char
 	return 0;
 }
 
+npmx_gpio_mode_t gpio_mode_convert(uint8_t mode)
```

Comment:
```suggestion
static npmx_gpio_mode_t gpio_mode_convert(uint8_t mode)
```
It should be applied to other functions.